Makefile

```diff
@@ -6,8 +6,7 @@ test-unit:
 	GO111MODULE=on go test -race -v $(go list ./... | grep -v /demo/)
 
 test-unit-cover:
-	GO111MODULE=on go test -v -coverprofile=coverage.txt -covermode=atomic -coverpkg=all $(go list ./... | grep -v /demo/)
-	GO111MODULE=on go test -v -coverprofile=democoverage.txt -covermode=atomic -coverpkg=./... ./demo
+	GO111MODULE=on go test -v -coverprofile=coverage.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
```

Comment:
did you mean to remove the demo tests here? This line explicitly removes the demo package from the list of packages to test...

Reply:
This make rule isn't being used in CI. i'm unconvinced the demo counts as a 'unit' test, perhaps the right answer is to remove this make rule entirely.

Reply:
No I think it makes sense to remove demo tests here - they are definitely not unit tests! Thanks for the explanation.